src/redux/actions/index.js

```diff
@@ -6,7 +6,7 @@ import {
   SET_CURRENT_CATEGORY,
   SET_LEARNT_PHRASES,
 } from '../constants';
-import {getAllCategories} from '../../data/dataUtils';
+import {getAllCategories as getLocalAllCategories} from '../../data/dataUtils';
```

Comment:
can you change this to getAllLocalCategories? The other way around is not valid english

Reply:
Yes please